.circleci/config.yml

```diff
@@ -69,21 +80,30 @@ workflows:
     version: 2
     build-test-deploy:
       jobs:
-        - build_and_test:
+        - build_and_test
+        - scan:
+              requires:
+                  - build_and_test
               filters:
                   tags:
                       only: /[0-9.]+(-(a|b|rc)[0-9]+)?/
+              context:
+                  - SonarCloud
         - deploy_snapshot:
               requires:
-                  - build_and_test
+                  - scan
               filters:
                   branches:
                       only: /((release|hotfix|support)/[0-9.]+(-(a|b|rc)[0-9]+)?|develop)/
+              context:
```

Comment:
we dont' actually need dockerhub here since we're not pushing to docker, may want it in mmsri